etcdmgmt/etcd-mgmt_test.go

```
@@ -19,3 +21,45 @@ func TestStartETCD(t *testing.T) {
 	tests.Assert(t, err == nil)
 	etcdCtx.Kill()
 }
+
+func TestWriteETCDPidFile(t *testing.T) {
+	cmd := exec.Command("etcd")
+	_ = cmd.Start()
```

you should not ignore error here.